localtypings/pxtarget.d.ts

```diff
@@ -880,6 +880,7 @@ declare namespace pxt.tutorial {
         templateCode?: string;
         autoexpandStep?: boolean; // autoexpand tutorial card if instruction text overflows
         metadata?: TutorialMetadata; // metadata about the tutorial parsed from the markdown
+        inlineHints?: boolean; // always show hint as part of the main card
```

Comment:
is this distinct from the @explicitHints metadata? specifying "explicit hints" indicates that the code should be parsed into the step instead of the hint

Reply:
Oh I didn’t know about that... will use

Reply:
Removing this option